src/services/auth-service.ts

```diff
@@ -0,0 +1,48 @@
+import {CreateUserRequest, LoginRequest} from "../global";
+import {PrismaClient} from '@prisma/client'
+
+const prisma = new PrismaClient()
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+const saltRounds = 10;
+
+export class AuthService {
+    registerUser = async (createUserRequest: CreateUserRequest) => {
+        const hashedPassword = await bcrypt.hash(createUserRequest.password, saltRounds);
```

Comment:
If you don't change them to parameters, we should be destructing `createUserRequest`
EG:
`const { firstName, email, password } = createUserRequest;`